src/dbnode/storage/series/buffer.go

```diff
@@ -246,9 +246,12 @@ func (b *dbBuffer) Write(
 	writeType := b.ResolveWriteType(timestamp, now)
 
 	if writeType == ColdWrite {
-		if !b.coldWritesEnabled {
-			return false, m3dberrors.ErrColdWritesNotEnabled
-		}
+		/*
+			Disable until cold writes is ready as this is confusing to users.
+			if !b.coldWritesEnabled {
+				return false, m3dberrors.ErrColdWritesNotEnabled
```

Comment:
We should still return an error here, otherwise a write that's too old will look like it got written successfully. How about just changing the error message to something like "datapoint is too far in the past or future"?

Reply:
But for folks who have it disabled (everyone), won't the writeType equal ColdWrite for all writes with timestamp outside of the buffer? By removing this, if we get an old time stamp, won't we return one of the errors in the following lines? (Granted there is a small window for race condition between the two checks, but I assume our buffers can accept some variability there). 

I can also just remove the writeType check completely, but the type is passed beyond this func.

Reply:
The checks in the following lines are the limits of ColdWrites and so checks using `retentionPeriod`/`futureRetentionPeriod`, instead of `bufferPast`/`bufferFuture`.

Reply:
Ah missed that. Ok, I'll just change the original error msg then.